src/pages/shared/components/SelectGasMenu/SelectGasMenuBody.tsx

```diff
@@ -98,14 +116,15 @@ const StyledNumericInput = styled(NumericInput)`
 	background-color: ${({ theme }) => theme.colors.accentL1};
 `;
 
-const DefinedGasSelector = styled.div<{ isActive: boolean }>`
+const DefinedGasSelector = styled.div<{ isActive: boolean; needsPadding?: boolean }>`
 	padding: 10px;
 	margin: 10px;
 	cursor: pointer;
 	display: flex;
 	justify-content: space-between;
 	color: ${(props) => props.theme.colors.fontTertiary};
 	${(props) => props.isActive && `background-color: ${props.theme.colors.accentL1}`};
+	${(props) => props.needsPadding && 'margin-top: 37px'};
```

Comment:
confirm this with Zoe, in previous floating messages, we were not shifting the UI to accomodate the message (which is why it's "floating" above the content, otherwise you wouldn't need it to be positioned this way).